course7/homeworks/anton/dc/complex_stack.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "complex.h"
+#include "stack"
```

Comment:
include ```<stack>```. Use angular brackets for 3rd party code, and quotes for local includes. Mostly by conventions these days.